src/overview.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Dart overview
+description: A short introduction to Dart
+---
+
+Dart is a client-optimized language for fast apps on any platform. Our goal is
+to offer the most productive programming language for multi-platform development,
+paired with a flexible execution runtime platform for app frameworks across
+Google.
+
+<img src="{% asset logo_lockup_dart_horizontal.png @path %}" width="300px" alt="Dart product logo">
```

Comment:
It looks a bit odd with the logo hanging here in the middle of the page. Perhaps we can pull it up into the heading of the page?

Reply:
Agree with you. I've floated it to the right now using my amazing CSS talents :)